src/BlockTypes/MiniCart.php

```diff
@@ -303,6 +303,9 @@ protected function get_cart_price_markup( $attributes ) {
 		$cart_controller     = $this->get_cart_controller();
 		$cart                = $cart_controller->get_cart_instance();
 		$cart_contents_total = $cart->get_subtotal();
+
+		if($cart->display_prices_including_tax() ) { $cart_contents_total += $cart->get_subtotal_tax();}
```

Comment:
The formatting of this `if` needs to be updated:
```suggestion
		if ( $cart->display_prices_including_tax() ) {
			$cart_contents_total += $cart->get_subtotal_tax();
		}
```